docs/minecraft/brigadier.md

```diff
@@ -29,6 +29,11 @@ per-argument validation, coloring and syntax hinting.
   <figcaption>Accurate Command Structure</figcaption>
 </figure>
 
+!!! warning "Alias Registration"
+
```

Comment:
The warning is fine, but I think the positioning is a little awkward. It's just kinda floating under the screenshots. Could we move it under the brig manager or something? :thinking:

Reply:
moved